lib/stat_tracker.rb

```diff
@@ -54,6 +54,39 @@ def highest_scoring_home_team
     end
   end
 
+  def highest_scoring_home_team
+    # @game_teams.csv
+    # 1. find total amount of goals scored by each home team and keep track of the amount of games
+    # hash {[team_id #] => [total goals, total games played by home team]}
```

Comment:
Same comments on this as for the other method because they are essentially the same.